GSA-TTS/FAC
# Sk/read as str,update error table

https://github.com/GSA-TTS/FAC/issues/2993 - Force string type for all fields read from csv.
https://github.com/GSA-TTS/FAC/issues/2977 - Rename MigrationErrorDetail fields.

Test:
- 2993
  - Load data using https://github.com/GSA-TTS/FAC/blob/main/backend/census_historical_migration/README.md#how-to-load-test-census-data-into-postgres
  - Run the following query to fetch the lengths of zip codes we have:
`select length(chme."ZIPCODE"), count(*)  from census_historical_migration_elecauditheader chme 
group by length(chme."ZIPCODE")`
  - Verify that this fix only produces zip codes of length 5 and 9. For example:
    - <img width="271" alt="Screenshot 2023-12-14 at 9 18 56 AM" src="https://github.com/GSA-TTS/FAC/assets/135276194/4e7fb308-ad6e-4fcf-a16d-dd456cbbfc6b">
- 2977
  - Check the fields in the MigrationErrorDetail table.
    - <img width="415" alt="Screenshot 2023-12-14 at 9 22 15 AM" src="https://github.com/GSA-TTS/FAC/assets/135276194/b8093718-5ad6-4e8b-a5be-436466ad0f52">



## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
